docs/swagger_v3/transactions.spec.yaml

```diff
@@ -212,8 +212,14 @@ paths:
           required: false
           schema:
             type: string
+        - name: ScopeParam
+          in: query
+          name: scope
+          description: 'Scopes results in a txi, generation or time(unix timestamp) range'
+          schema:
+              type: string
+              pattern: '(time|txi|gen):\d+(-\d+)?'
```

Comment:
`txi` is no longer allowed in v3, the concept of txi is not exposed to the user

Reply:
Right, i coppied the scope definition, I'll remove it there as well